content/hi/docs/contribute/review/reviewing-prs.md

```diff
@@ -0,0 +1,150 @@
+---
+title: पुल अनुरोधों की समीक्षा करना
+content_type: concept
+main_menu: true
+weight: 10
+---
+
+<!-- अवलोकन -->
+
+कोई भी प्रलेखन पुल अनुरोध की समीक्षा कर सकता है। खुले पुल अनुरोधों को देखने के लिए
+कुबेरनेट्स वेबसाइट रिपॉजिटरी में [पुल अनुरोध](https://github.com/kubernetes/website/pulls) अनुभाग पर जाएँ।
+
+
+प्रलेखन पुल अनुरोधों की समीक्षा करना कुबेरनेट्स समुदाय से अपना परिचय कराने का एक शानदार तरीका
+है. यह आपको कोड बेस सीखने और अन्य योगदानकर्ताओं के साथ विश्वास बनाने में मदद करता है।
+
+समीक्षा करने से पहले, यह एक अच्छा विचार है:
+
+- [विषय मार्गदर्शिका](/docs/contribute/style/content-guide/) और [शैली मार्गदर्शिका](/docs/contribute/style/style-guide/) पढ़ें ताकि आप सूचित टिप्पणियाँ लिख सकें।
+- कुबेरनेट्स प्रलेखन समुदाय में विभिन्न [भूमिकाओं और जिम्मेदारियों](/docs/contribute/participate/roles-and-responsibilities/) को समझें।
+
+<!-- body -->
+
+## शुरू करने से पहले
+
+समीक्षा शुरू करने से पहले:
+
+- [CNCF आचार संहिता](https://github.com/cncf/foundation/blob/main/code-of-conduct.md) पढ़ें और सुनिश्चित करें कि आप हर समय इसका पालन करते हैं।
+- विनम्र, विचारशील और मददगार बनें।
+- PR के सकारात्मक पहलुओं के साथ-साथ परिवर्तनों पर टिप्पणी करें।
+- आपकी समीक्षा कैसे प्राप्त हो सकती है, इस बारे में सहानुभूतिपूर्ण और सावधान रहें।
+- अच्छी मंशा मानें और स्पष्ट प्रश्न पूछें।
+- अनुभवी योगदानकर्ता, नए योगदानकर्ताओं के साथ जोड़ी बनाने पर विचार करें जिनके कार्य में व्यापक परिवर्तन की आवश्यकता है।
+
+## पुनरावलोकन प्रक्रिया
+
+सामान्य तौर पर, अंग्रेज़ी में सामग्री और शैली के लिए पुल अनुरोधों की समीक्षा करें। चित्र 1 
+समीक्षा प्रक्रिया के चरणों को रेखांकित करता है। प्रत्येक चरण का विवरण इस प्रकार है।
+
+<!-- See https://github.com/kubernetes/website/issues/28808 for live-editor URL to this figure -->
+<!-- You can also cut/paste the mermaid code into the live editor at https://mermaid-js.github.io/mermaid-live-editor to play around with it -->
+
+{{< mermaid >}}
+flowchart LR
+    subgraph fourth[समीक्षा शुरू करें]
+    direction TB
+    S[ ] -.-
+    M[टिप्पणियाँ लिखें] --> N[परिवर्तनों की समीक्षा करें]
+    N --> O[नए योगदानकर्ताओं को <br>टिप्पणी का चयन करना चाहिए]
+    end
+    subgraph third[PR का चयन करें]
+    direction TB
+    T[ ] -.-
+    J[विवरण<br>और टिप्पणियाँ पढ़ें]--> K[Netlify प्रीव्यू बिल्ड <br> के साथ अपने परिवर्तनों का पूर्वावलोकन करें]
+    end
+ 
+  A[खुली PR सूची की समीक्षा करें]--> B[खुले PR को <br>लेबल द्वारा फ़िल्टर करें]
+  B --> third --> fourth
+     
+
+classDef grey fill:#dddddd,stroke:#ffffff,stroke-width:px,color:#000000, font-size:15px;
+classDef white fill:#ffffff,stroke:#000,stroke-width:px,color:#000,font-weight:bold
+classDef spacewhite fill:#ffffff,stroke:#fff,stroke-width:0px,color:#000
+class A,B,J,K,M,N,O grey
+class S,T spacewhite
+class third,fourth white
+{{</ mermaid >}}
```

Comment:
This portion is getting printed as normal text, do necessary modifications in it

Reply:
Please check how the Chinese/Korean localizations have performed these changes and replicate it here with the Hindi script.